pandas/core/groupby/groupby.py

```diff
@@ -992,75 +988,66 @@ def __getattribute__(self, attr: str):
             return super().__getattribute__(attr)
 
     @final
-    def _make_wrapper(self, name: str) -> Callable:
-        assert name in self._apply_allowlist
-
+    def _op_via_apply(self, name: str, *args, **kwargs):
+        """Compute the result of an operation by using GroupBy's apply."""
+        f = getattr(type(self._obj_with_exclusions), name)
         with self._group_selection_context():
             # need to setup the selection
             # as are not passed directly but in the grouper
-            f = getattr(self._obj_with_exclusions, name)
-            if not isinstance(f, types.MethodType):
-                #  error: Incompatible return value type
-                # (got "NDFrameT", expected "Callable[..., Any]")  [return-value]
-                return cast(Callable, self.apply(lambda self: getattr(self, name)))
+            f = getattr(type(self._obj_with_exclusions), name)
+            if not callable(f):
```

Comment:
Just curious, what `name`s would hit this path?

Reply:
Properties - plot, is_monotonic_increasing, is_monotonic_decreasing, dtype, and dtypes.